test/NuGet.Clients.Tests/NuGet.CommandLine.Test/compiler/resources/runtime.json

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
This file is in two test projects. Would it be worthwhile moving to Test.Utility, so there's only one copy? The file contents is not specific to either test project.

Reply:
Sure, I'll look into it.